src/main/UseCases/GetReview.java

```diff
@@ -0,0 +1,19 @@
+package UseCases;
+
+import java.util.ArrayList;
+import Constants.*;
+import Entities.Recipe;
+import Entities.Review;
+import Entities.User;
+
+public class GetReview{
+    public ArrayList<Review> fromUser (String username){
```

Comment:
Could use documentation to make methods purpose clearer to viewer

Reply:
will do later, ty!!